src/Converter.cpp

```diff
@@ -123,13 +125,22 @@ int convertStreams(
                 << " samples" << std::endl;
 
         const int realEnd = computeRangeEnd(start, end, stream->getSize(), true);
-        const int exportedSize = realEnd - start;
-
+        int exportedSize = realEnd - start;
+        
+        if (exportedSize < 0) {            
```

Comment:
I'd like to keep the coding style consistent in this repository: move `{` to the beginning of the next line.